packages/core/src/icon/icon.scss

```diff
@@ -1,8 +1,21 @@
+$sizes: sm, md, lg;
+
 :host {
   --fill-default: currentColor;
   --fill-accent: rgb(var(--sl-color-palette-accent-300));
-  
-  display: inline-flex;
-  height: 2rem;
-  width: 2rem;
+   
+  display: grid;
```

Comment:
Not sure about this one; this will cause the icon component so have the full width of the parent? Shouldn't it be `inline-grid`?